Automattic/wp-calypso
# DIFM: Adds more fields to the Contact page

### Changes proposed in this Pull Request

* This PR adds new fields to the Contact page section of the Website Content step in the DIFM post-checkout flow.
* These new fields are hidden behind a feature flag: `signup/redesigned-difm-flow` 
<img width="811" alt="image" src="https://user-images.githubusercontent.com/5436027/161497607-cd6f7497-52cc-42fb-b89b-5dde7a0e514d.png">


### Testing instructions



#### Testing the new contact page fields
* Apply D78088-code
* Go to `/start/do-it-for-me?flags=signup/redesigned-difm-flow` and complete checkout.
* On the Website Content step, append `&flags=signup/redesigned-difm-flow` to the URL.
* Open the Contact page section. Confirm that you can see the extra fields: Email, Phone and Address.
* Type some text in each of the new fields and also in the Content field. Refresh the page. Confirm that the typed text is preserved.
* Submit the step. Further test instructions are in D78088-code



Related to 0-as-1202010833393733/f
